src/slack_insights/gui/app.py

```
@@ -0,0 +1,245 @@
+		with results_container:
+			with ui.row().classes("w-full justify-center items-center gap-4 p-8"):
+				ui.spinner(size="lg")
+				ui.label("Searching...").classes("text-gray-600")
+
+		# Execute query
+		if query_engine:
+			# Use real backend
+			try:
+				results = query_engine.execute_query(query_text)
+				ui.timer(0.1, lambda: display_results(results_container, results), once=True)
+			except Exception as e:
```

Ruff (0.13.3)

Do not catch blind exception: `Exception`

(BLE001)